lib/galaxy/jobs/runners/drmaa.py

```diff
@@ -316,7 +316,15 @@ def stop_job( self, job ):
                 self.ds.kill( ext_id )
             else:
                 # FIXME: hardcoded path
-                subprocess.Popen( [ '/usr/bin/sudo', '-E', kill_script, str( ext_id ), str( self.userid ) ], shell=False )
+
+                command = [ '/usr/bin/sudo', '-E', ]
+                for v in ["PATH", "LD_LIBRARY_PATH", "PKG_CONFIG_PATH"]:
```

Comment:
Again, I don't understand why this is necessary, 'sudo -E' should really preserve the environment.